stdlib/public/Concurrency/Actor.swift

```diff
@@ -19,6 +19,20 @@ import Swift
 /// implicitly conform to this protocol.
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 public protocol Actor: AnyObject, Sendable {
+
+  /// Retrieve the executor for this actor as an optimized, unowned
+  /// reference.
+  ///
+  /// This property must always evaluate to the same executor for a
+  /// given actor instance, and holding on to the actor must keep the
+  /// executor alive.
+  ///
+  /// This property will be implicitly accessed when work needs to be
+  /// scheduled onto this actor.  These accesses may be merged,
+  /// eliminated, and rearranged with other work, and they may even
+  /// be introduced when not strictly required.  Visible side effects
+  /// are therefore strongly discouraged within this property.
+  var unownedExecutor: UnownedSerialExecutor { get }
```

Comment:
yay, happy to see the Actor protocol put to good use with those :) 

And we're following the same pattern in distributed :)

// Complete nitpick for future: I was very very tempted to make those properties be $executor, $transport, $address... to not conflict with user defined things... Would it be weird to define those with `$`? They're _kind of_ synthesized after all... (Allowing this would need to make parsing more permissive, perhaps for some internal marked known names etc hm...)

Reply:
I deliberately didn't call it `executor` because I wanted to leave space for an owned property (presumably implemented in terms of this one).  I'm not worried about actor types wanting to name something `unownedExecutor`.

Reply:
I see; what would the owned version be preferable for? (I may be missing things to understand the difference here, sorry)